src/workloads/contrib/analysis/test_result_summary.py

```diff
@@ -77,9 +83,17 @@ def replace_suffix(string, suffixTarget, newSuffix):
     suffix_start = string.rfind(suffixTarget)
     return string[0:suffix_start] + newSuffix
 
+def animate(interrupt_event):
```

Comment:
It turned out to be much slower to do it this way than to use the CSV format... but rather than fix that I just added a loading bar... It still doesn't take more than a handful of seconds which I think is acceptable. I was chatting with @naama-bareket and I do think we could do better by directly processing the '.ftdc' file, but that would be a more major re-write since it's a BSON-direct file format so we'd be messing with bits more and need to use some bson library.

Reply:
Yep, I think a handful of seconds is find given the complexity of working with .ftdc directly. And a loading bar will keep us patient :)

Reply:
Agreed!